setup.py

```diff
@@ -13,7 +13,7 @@
 
 setup(
     name='dicom_numpy',
-    version='0.2.0',
+    version='0.3.0',
     description='Extract image data into a 3D numpy array from a set of DICOM files.',
     long_description=long_description,
     url='https://github.com/innolitics/dicom-numpy',
```

Comment:
@ReeceStevens one other small change request: perhaps we should change line 24 from "1 - Planning" to one of the other development statuses?

https://pypi.org/classifiers/

I am not sure which one makes sense. Maybe `Development Status :: 5 - Production/Stable`?

Reply:
I think `Development Status :: 5 - Production/Stable` makes sense; it's the same label that's used for PyDicom. I'll update it